cuenca_validations/types/requests.py

```
@@ -505,6 +524,13 @@
             raise ValueError('The total percentage should be 100%')
         return beneficiaries
 
+    @field_validator('profession')
+    @classmethod
+    def validate_profession(cls, profession: Profession) -> Profession:
+        if profession == Profession.otro:
+            raise ValueError('Profession "otro" is not allowed for this model')
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)